try{console.log(Hello)}
catch(err){
}